bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractOptionsFieldImpl.java

```diff
@@ -50,6 +50,10 @@ public abstract class AbstractOptionsFieldImpl extends AbstractFieldImpl impleme
     @Nullable
     protected String[] enumNames;
 
+    @ValueMapValue(injectionStrategy = InjectionStrategy.OPTIONAL, name = ReservedProperties.PN_MULTI_DEFAULT_VALUES)
+    @Nullable
+    protected Object[] multiDefaultValues;
```

Comment:
Can we add a code comment here, that this property would only exist for forms created in UE, also add reasoning on why this is done